setup.py

```diff
@@ -168,7 +168,7 @@ def get_exts_for(name):
         "funcsigs>=1.0.0; python_version=='2.7'",
         "typing; python_version<'3.5'",
         "packaging>=17.1",
-        "protobuf>=3",
+        "protobuf>=3,<3.18",
```

Comment:
```suggestion
        "protobuf>=3,<3.18; python_version<'3.6'",
        "protobuf>=3; python_version>='3.6'",
```

This way only people on 2.7 and 3.5 get <3.18 and everyone else gets the latest